pyvrp/cpp/Individual.h

```diff
@@ -85,28 +83,22 @@ class Individual
      *
      * @param data           Data instance describing the problem that's being
      *                       solved.
-     * @param penaltyManager Penalty manager, used to compute the objective.
      * @param rng            Random number generator.
      */
-    Individual(ProblemData const &data,
-               PenaltyManager const &penaltyManager,
-               XorShift128 &rng);
+    Individual(ProblemData const &data, XorShift128 &rng);
 
     /**
      * Constructs an individual having the given routes as its solution.
      *
      * @param data           Data instance describing the problem that's being
      *                       solved.
-     * @param penaltyManager Penalty manager, used to compute the objective.
      * @param routes         Solution's route list.
      */
-    Individual(ProblemData const &data,
-               PenaltyManager const &penaltyManager,
-               Routes routes);
-};
+    Individual(ProblemData const &data, Routes routes);
 
-// Outputs an individual into a given ostream in VRPLIB format
-std::ostream &operator<<(std::ostream &out, Individual const &indiv);
+    // Outputs an individual into a given ostream in VRPLIB format
+    friend std::ostream &operator<<(std::ostream &out, Individual const &indiv);
```

Comment:
Suggestion: can we move this friend declaration up to the `private` section at the start of the class declaration? I think that works, and then the friendness is just an implementation detail again.